### ISSUE TITLE
this.typed is undefined

### ISSUE THREAD

**Issue by Queatz** [thevladeffect/ngx-typed-js#20]
https://github.com/thevladeffect/ngx-typed-js/blob/062ee2c5ac4a6fcb162e5a08e71767a46c715e24/projects/ngx-typed-js/src/lib/ngx-typed-js.component.ts#L133

I get this with the following code:

```ts
<ngx-typed-js [strings]="['Find', 'Create', 'Share', 'Automate']">
        <span class="font-accent typing"></span> trading strategies
</ngx-typed-js>
```

**Comment by himanshusaraswat** [comment]
Facing the same issue, when upgraded to NG12 today, was working fine with NG -9. 
PS: I am using multiple <ng-types-js> tags which are rendered when the previous tag emits "completed" event.

**Comment by kylanee** [comment]
See also #21, it could be the same problem.

**Comment by thevladeffect** [comment]
This should be fixed in 2.1.1
